app/utils/hentFraApi.ts

```diff
@@ -0,0 +1,13 @@
+import søkerMock from '~/mock/søkerMock';
+
+export const hentSøkerFornavn = () => {
+  const søker = søkerMock;
+  //her skal vi fetche fra API
```

Comment:
Her burde vi gjøre litt som med sanitytekstene at vi henter dataene også legger vi dem i en context som vi henter fra hver gang vi skal bruke dem. Dersom vi skal gjøre et apikall hver gang vi trenger data om søker kan potensielt bli mange apikall.

Reply:
Ny oppgave lagt til for dette: https://trello.com/c/qnrGdGHo/54-fikse-at-api-kall-skjer-n%C3%A5r-det-trengs-og-lagre-bruker-data-i-en-context